Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1051,6 +1051,23 @@ func _vjpRelu<T: TensorFlowFloatingPoint>(
     (relu(x), { v in Tensor(x .> 0) * v })
 }
 
+/// Returns the Gaussian Error Linear Unit (GELU) of the specified tensor element-wise.
```

Comment:
```suggestion
/// Returns a tensor by applying the Gaussian Error Linear Unit (GELU) to the specified tensor element-wise.
```

Also, if you are adding activation functions which aren't extremely popular, could you also add a link to the paper this was introduced in?

Reply:
@jon-tow you seem to have reversed this change

Reply:
@Shashi456 It brought line past the 100 column limit. I didn't realize it until after the commit.Sorry about that.

Reply:
@jon-tow, It's absolutely alright. You can even do something like 
```
/// Returns a tensor by applying the Gaussian Error Linear Unit (GELU) 
/// to the specified tensor element-wise.
```
But your current wording is alright as well.

Reply:
@Shashi456 I thought your suggestion was worded well. I ended up using a different wording that managed to keep the line under 100 columns (most recent commit here). Appreciate the suggestions!